packages/experiments/src/components/MicroFeedback/MicroFeedback.base.tsx

```diff
@@ -39,57 +40,50 @@ export class MicroFeedbackBase extends React.Component<IMicroFeedbackProps, IMic
     });
 
     return (
-      <Stack className={this.classNames.root} horizontal>
-        <div ref={this.likeRef}>
-          <IconButton menuIconProps={{ iconName: likeIcon }} title={this.props.thumbsUpTitle} onClick={this._likeVote} />
-        </div>
-        <div ref={this.dislikeRef}>
-          <IconButton menuIconProps={{ iconName: dislikeIcon }} title={this.props.thumbsDownTitle} onClick={this._dislikeVote} />
-        </div>
-        {this.props.thumbsUpQuestion ? (
-          <Callout
-            className={this.classNames.followUpContainer}
-            hidden={hideThumbsUpCallout}
-            role="alertdialog"
-            gapSpace={0}
-            target={this.likeRef.current}
-            setInitialFocus={true}
-            onDismiss={this._onCalloutDismiss}
-          >
-            <FocusZone direction={FocusZoneDirection.vertical}>
-              <Text block={true} className={this.classNames.followUpQuestion} variant="small">
-                {this.props.thumbsUpQuestion.question}
-              </Text>
-              <List
-                items={this.props.thumbsUpQuestion.options}
-                className={this.classNames.followUpOptionText}
-                onRenderCell={this._onRenderCalloutItem}
-              />
-            </FocusZone>
-          </Callout>
-        ) : null}
-        {this.props.thumbsDownQuestion ? (
-          <Callout
-            className={this.classNames.followUpContainer}
-            hidden={hideThumbsDownCallout}
-            role="alertdialog"
-            gapSpace={0}
-            target={this.dislikeRef.current}
-            setInitialFocus={true}
-            onDismiss={this._onCalloutDismiss}
-          >
-            <FocusZone direction={FocusZoneDirection.vertical}>
-              <Text block={true} className={this.classNames.followUpQuestion} variant="small">
-                {this.props.thumbsDownQuestion.question}
-              </Text>
-              <List
-                items={this.props.thumbsDownQuestion.options}
-                className={this.classNames.followUpOptionText}
-                onRenderCell={this._onRenderCalloutItem}
-              />
-            </FocusZone>
-          </Callout>
-        ) : null}
+      <Stack className={this.classNames.root}>
+        <Stack horizontal className={this.classNames.iconContainer}>
+          {this.props.children}
```

Comment:
Hmm, this looks like the overlay approach? I thought you said this would be too limiting and were going to pursue the render approach instead?

Reply:
Perhaps I misunderstood? I thought overlay meant using the Overlay component.